ui.frontend/src/main/webpack/site/_variables.scss

```diff
@@ -2,6 +2,7 @@
 //== Font
 
 $font-family:           'Montserrat', sans-serif;
+$font-size:             16px;
```

Comment:
Should we keep with the 'rem' version?

Reply:
i am guessing auto merge added both. i converted px to rem, so just rem should be fine. Will make that change.

Reply:
px version removed in next commit